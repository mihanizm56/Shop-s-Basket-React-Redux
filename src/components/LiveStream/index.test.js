import React from "react"
import { shallow } from "enzyme"
import { LiveStream } from "."

describe("LiveStream", () => {
  const component = shallow(<LiveStream />)

  it("Renders correct", () => {
    expect(component).toMatchSnapshot()
  })
})
