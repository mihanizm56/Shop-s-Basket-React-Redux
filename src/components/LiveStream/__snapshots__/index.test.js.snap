// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditNumber Renders correct 1`] = `
<styled.div>
  Смотрите наш прямой эфир
</styled.div>
`;

exports[`LiveStream Renders correct 1`] = `
<styled.div>
  Смотрите наш прямой эфир
</styled.div>
`;
